src/Main.purs

```diff
@@ -216,18 +216,25 @@ main = void do
     | otherwise =
        throwException (Child.toStandardError err)
 
+  isEmptySpan filename pos =
+    filename == "" ||
```

Comment:
Another filename case here would be `"<module>"`, although I think the pos guarding will cover the cases that arises in too.

Reply:
Why would "<module>" be a file name?

Reply:
https://github.com/purescript/purescript/issues/3417

Reply:
As for actually why, I don't know, it's undoubtedly something stupid I did though :smile:

Reply:
There are too many possible cases to count and guard against 😭 :rage4:

Reply:
Yeah - it is the compiler's fault anyway. Especially given sometimes these error message have the module information, but still lack a span.